src/main/java/net/minecraftforge/registries/DataPackRegistriesHooks.java

```diff
@@ -57,4 +58,13 @@ public static Set<ResourceKey<? extends Registry<?>>> getSyncedCustomRegistries(
     {
         return SYNCED_CUSTOM_REGISTRIES_VIEW;
     }
+
+    /**
+     *  FORGE: add non-vanilla registry namespace to loader directory, same format as tag directory (see net.minecraft.tags.TagManager#getTagDir(ResourceKey))
+     * @param registryKey original registry key
+     * @return path of the registry key. Prefixed with the modid if it wasn't minecraft
+     */
+    public static String prefixNamespace(ResourceLocation registryKey) {
+        return registryKey.getNamespace().equals("minecraft") ? registryKey.getPath() : registryKey.getNamespace() +  "/"  + registryKey.getPath();
```

Comment:
This is not the same behavior as the old patch. Please fix.

Reply:
I see why this was changed from the discussion in #github-discussions. For anyone else here: Using RegistryManager#FROZEN was only necessary on 1.18.2 to filter out datapack registries that were not made through Forge's system to preserve their datapack format and behavior. It was decided that everyone on 1.19+ should be using the forge system, so this is reasonable to remove.